ModulePresentationsForCAP/gap/ModulePresentationFunctors.gi

```diff
@@ -385,15 +393,15 @@ InstallMethod( FunctorDoubleDualLeft,
      
            function( object )
            
-             return ApplyFunctor( dual_functor, ApplyFunctor( dual_functor, object ) );
+             return ApplyFunctor( dual_functor, Opposite( ApplyFunctor( dual_functor, Opposite( object ) ) ) );
```

Comment:
You could simply use PreCompose for functors in CAP in order to define the double dual.

Reply:
FunctorDoubleGradedDualLeft = PreCompose( FunctorGradedDualLeft, ( FunctorGradedDualLeft )^op )

But there is yet no method in Cap to compute the opposite of a functor.

Reply:
It probably would make more sense to add this method. But this can be done in a separate PR.